common/src/main/kotlin/streams/utils/Neo4jUtils.kt

```diff
@@ -37,6 +37,21 @@ object Neo4jUtils {
         }
     }
 
+    fun hasApoc(db: GraphDatabaseAPI): Boolean {
+        try {
+            return db.execute("RETURN apoc.version() AS version") {
+                it.columnAs<String>("version").next()
+                true
+            }
+        } catch (e: QueryExecutionException) {
+            if (e.statusCode.equals("Neo.ClientError.Statement.SyntaxError", ignoreCase = true)
```

Comment:
too specific
I would return false in any error case

Reply:
done